pkg/ucglib/patches/0001-add-riot-makefiles.patch

```diff
@@ -0,0 +1,62 @@
+From 6bdd654ab1b904d245c57765262e1f6736224ddb Mon Sep 17 00:00:00 2001
```

Comment:
I think this patch could be replaced by Makefiles in the pkg directory that are copied to the ucglib build directory, see semtech-loramac package for an example. It will make future maintenance easier.

Reply:
Fixed.